### PR TITLE
CATROID-1393 Remove redundant code of directory-import

### PR DESCRIPTION
**Pull request by Walcho1125**
*Please enter a short description of your pull request and add a reference to the Jira ticket.*
https://jira.catrob.at/browse/CATROID-1393 
Remove redundant code of directory-import
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [ ] Perform a self-review of the changes
- [ ] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by bengaal — APPROVED** [review]
Looks good to me! Thank you!

File: catroid/src/main/java/org/catrobat/catroid/io/asynctask/ProjectUnZipperAndImporter.kt [hunk 1]
```diff
@@ -63,8 +69,56 @@ private fun unzipAndImportProject(projectDir: File): Boolean = try {
         StorageOperations.deleteDir(cachedProjectDir)
     }
     ZipArchiver().unzip(projectDir, cachedProjectDir)
-    ProjectImportTask.task(listOf(cachedProjectDir))
+    importProject(cachedProjectDir)
 } catch (e: IOException) {
     Log.e(TAG, "Cannot unzip project " + projectDir.name, e)
```
Comment (juliajulie95) [inline catroid/src/main/java/org/catrobat/catroid/io/asynctask/ProjectUnZipperAndImporter.kt:74]: unzip

File: catroid/src/main/java/org/catrobat/catroid/io/asynctask/ProjectUnZipperAndImporter.kt [hunk 2]
```diff
@@ -27,9 +27,15 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
```
Comment (juliajulie95) [inline catroid/src/main/java/org/catrobat/catroid/io/asynctask/ProjectUnZipperAndImporter.kt:27]: rename file

File: catroid/src/main/java/org/catrobat/catroid/io/asynctask/ProjectUnZipperAndImporter.kt [hunk 3]
```diff
@@ -63,8 +69,56 @@ private fun unzipAndImportProject(projectDir: File): Boolean = try {
         StorageOperations.deleteDir(cachedProjectDir)
```
Comment (juliajulie95) [inline catroid/src/main/java/org/catrobat/catroid/io/asynctask/ProjectUnZipperAndImporter.kt:69]: rename function

**Review by juliajulie95 — APPROVED** [review]

**Pull request merged**